tests/test_classdecorators.py

```diff
@@ -911,3 +911,25 @@ def step(self, x: int) -> int:
     DBOS.launch()
 
     assert inst.step(input) == input + input
+
+
+def test_class_with_only_steps(dbos: DBOS) -> None:
+
+    class StepClass:
+        def __init__(self, x: int) -> None:
+            self.x = x
+
+        @DBOS.step()
+        def step(self, x: int, expr: Callable[[int, int], int]) -> int:
+            return expr(self.x, x)
+
+    input = 5
+    inst = StepClass(5)
+
+    l = lambda x, y: x + y
+
+    @DBOS.workflow()
+    def test_workflow() -> int:
+        return inst.step(input, l) + inst.step(input, l)
+
+    assert test_workflow() == input * 4
```

Comment:
Do we want to test OAOO here? Because we want to make sure the step is properly registered.

Reply:
Steps don't need to be registered (unless they're enqueued, which will fail here, but we're not testing that)

Reply:
No they don't, but how do we know the functions are properly wrapped?